Agents/utils/python-utils/setup.py

```diff
@@ -0,0 +1,24 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="twautils",
+    version="1.0.0",
+    author="CMCL Innovations",
+    author_email="support@cmclinnovations.com",
+    description="TheWorldAvatar Utilities",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/cambridge-cares/TheWorldAvatar",
+    packages=['twautils', 'twautils.log'],
+    package_data={"": ["*.conf"]},
+    classifiers=(
+        "Programming Language :: Python :: 3",
+        "Operating System :: OS Independent",
+    ),
```

Comment:
Looks like this needs to be a list rather than a tuple; in older versions of setuptools it raises an error, in newer versions I think it just shows a warning
.
```suggestion
    classifiers=[
        "Programming Language :: Python :: 3",
        "Operating System :: OS Independent",
    ],
```

Reply:
Changed.